src/config.ts

```diff
@@ -49,7 +49,7 @@ export interface IntegrationConfig extends IntegrationInstanceConfig {
   /**
    * For use in tests only. Avoid ingesting entire account's worth of applications to reduce PollyJS Recording size
    */
-  targetApplication?: string;
+  targetVeracodeApplication?: string;
```

Comment:
nit:  Did you mean to change this and the above comment to read 'assessment' instead?

Reply:
In the [veracode api](https://app.swaggerhub.com/apis/Veracode/veracode-applications_api_specification/1.0#/Application%20information%20API/getApplicationsUsingGET) we are pulling in what they denote as an `Application`. However, that entity class is primarily used in J1 as an anchor point in the graph connecting various subgraphs sourced from different integration definitions.

So throughout the project when I refer to Application, I am specifically mentioning the VeracodeApplication as represented in their API. When we eventually represent this data from Veracode as an Entity with `_class=Assessment`, we switch to the `Assessment` naming convention.

Reply:
I allude to this [here](https://github.com/JupiterOne/graph-veracode/pull/20/files#diff-c54113cf61ec99691748a3890bfbeb00e10efb3f0a76f03a0fd9ec49072e410aR1), but I am happy to leave a more descriptive comment in that file over the `Application` type. Which should perhaps be renamed to `VeracodeApplication` now that I think more about it.